tests/quick-search.spec.ts

```diff
@@ -9,18 +9,24 @@ const openQuickSearchByShortcut = async (page: Page) =>
   await withCtrlOrMeta(page, () => page.keyboard.press('k', { delay: 50 }));
 
 async function assertTitle(page: Page, text: string) {
-  const locator = page.locator('.affine-default-page-block-title').nth(0);
-  const actual = await locator.inputValue();
-  expect(actual).toBe(text);
+  const edgeless = page.locator('affine-edgeless-page');
+  if (!edgeless) {
```

Comment:
```suggestion
  const edgeless = page.locator('affine-edgeless-page');
  if (!edgeless) {
```

I'm not very familiar with the testing of this project, will this work?

Reply:
no

Reply:
you need `await page.locator('affine-edgeless-page').toHaveCount(0);`